libmachine/drivers/base.go

```diff
@@ -11,11 +13,18 @@ const (
 	DefaultEngineInstallURL = "https://get.docker.com"
 )
 
+// TODO: add additional machineOS values here to support them
+var SupportedMachineOS = map[string]bool{
```

Comment:
youre properly using this everywhere with ok checks so you can convert to a `map[string]struct{}{}` if you want to save a couple bits.

Reply:
I would like this better if it wasn't exported. Exported package-level variables can be changed anywhere. I would like this to be unexported with an exported "check" function.

I also like the idea of using `map[string]struct{}` here.

Reply:
addressed - please review when you have a few minutes